### PR TITLE
style: Update design for balance card in assets portfolio redesign

### PR DESCRIPTION
**Pull request by wal33d006**

### CODE REVIEW

File: packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart [hunk 1]
```diff
@@ -14,30 +13,36 @@ class BalanceCard extends StatelessWidget {
...
-        subtitle: Text(amountDisplayText),
-        // TODO: Fix this during refactor
-        leading: icons[Random().nextInt(2)],
-        trailing: ElevatedButton(
-          style: ElevatedButton.styleFrom(
-            shape: const StadiumBorder(),
+    return ListTile(
+      title: Text(
+        denomText,
+        style: const TextStyle(fontWeight: FontWeight.w600),
```
Comment (andrzejchm) [inline packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart:21]: lets mark a todo here stating:
```dart
// TODO use `CosmosTheme.of(context).textStyles.title0Medium` when implemented
```

File: packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart [hunk 2]
```diff
@@ -14,30 +13,36 @@ class BalanceCard extends StatelessWidget {
...
+      trailing: Column(
+        crossAxisAlignment: CrossAxisAlignment.end,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(
+            amountDisplayText,
+            style: TextStyle(
+              fontWeight: FontWeight.w600,
+              fontSize: CosmosTheme.of(context).fontSizeS,
+            ),
```
Comment (andrzejchm) [inline packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart:37]: same here, lets add a todo :
```dart
// TODO use `CosmosTheme.of(context).textStyles.title1Medium` when implemented
```

**Review by andrzejchm — CHANGES_REQUESTED** [review]
👏🏻 let's add few todos, and we're good to go

File: packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart [hunk 3]
```diff
@@ -1,8 +1,7 @@
-import 'dart:math';
-
 import 'package:cosmos_ui_components/cosmos_theme.dart';
 import 'package:flutter/material.dart';
 
+// TODO: Add a parameter for icon
```
[the remark this answers is not in the document]
Comment (wal33d006) [reply packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart:4]: Updated

File: packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart (hunk 1 above)
Comment (wal33d006) [reply packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart:22]: Updated

File: packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart (hunk 2 above)
Comment (wal33d006) [reply packages/cosmos_ui_components/lib/components/template/cosmos_balance_card.dart:39]: Updated

**Review by andrzejchm — APPROVED** [review]

**Pull request merged**